examples/stm32f469_discovery/game_of_life/project.xml

```diff
@@ -1,5 +1,5 @@
 <library>
-  <extends>modm:disco-f469ni</extends>
+  <extends>modm:disco-f469ni/revB03</extends>
```

Comment:
Before merging this we should also fix the actual issue with the B-03 revision. 😉

Reply:
Sure, I just wanted to prototype this idea quickly to get a feel for it.

Reply:
Well… do you think this is a good idea? This is basically just a shortcut for the hidden repo option…